lib/hz2600/Kazoo/Api/Collection/Accounts.php

```diff
@@ -14,6 +17,69 @@ public function fetch(array $filter = array()) {
     }
 
     /**
+     * POST request to create a new icon
+     * $type = icon / logo
+     */
+    public function image($body, $type) {
+        // must be an x-base64 content type
+        $uri = $this->getURI('/whitelabel/'.$type);
+        $response = $this->getSDK()->post($uri, $body, array('Content-Type' => 'application/x-base64'));
+
+        $this->setCollection($response->getData());
+
+        $element_wrapper = $this->getElementWrapper();
+        $element_wrapper->unfetchable();
+
+        return $this;
+    }
+
+    /**
+     * PUT request to create a new whitelabel doc
+     *
+     */
+    public function whitelabel_create($body) {
```

Comment:
For consistency with the project, these function names should all be camelCase, not snake_case.